INSTALLATION.md

```diff
@@ -0,0 +1,91 @@
+# Atomkraft Installation Instructions
+
+Installing Atomkraft is as easy as `pip install atomkraft`.
+However, atomkraft depends on some software:
+your mileage may vary based on how much of that software you already have installed.
+In the following sections we describe how to install and what features are supported by different dependencies.
+If you are looking for a copy&paste solution to get Atomkraft up and running as quickly as possible,
+check the [TL;DR section](#tldr).
+
+## Installing Atomkraft
+
+Installing Atomkraft requires [Python](https://www.python.org) 3.10.
+
+If you don't have Python 3.10 already installed, please consult the [installation instructions](https://realpython.com/installing-python/) for different installation options.
+One quick way for installing and managing different Python options for your system is by using [pyenv](https://github.com/pyenv/pyenv) and running
+
+```
+pyenv install 3.10.5
+pyenv global 3.10.5
+```
+
+These two commands set the `python` version to `3.10.5`.
+If you want to go back to your old Python version, run `pyenv global system`.
+
+Once Python 3.10 installed, install Atomkraft with `pip install atomkraft`.
+
+That's it! Please verify that the tool is working by issuing `atomkraft` on the command line.
+You should see something like this:
+
+![Atomkraft CLI](docs/images/cli.png)
```

Comment:
It is a console output. why not just copy-paste the text from console in README?